### PR TITLE
correct yanking of URLs using the ';' separator

### PR DESCRIPTION
**Pull request by HolySmoke86**
as reported in #1987

### CODE REVIEW

**Comment by The-Compiler** [comment]
The OS X failures seem to be unrelated FWIW

**Comment by HolySmoke86** [comment]
Looks unrelated to me as well, though I only see WebEngine fail, not OSX :S

File: qutebrowser/browser/commands.py [hunk 1]
```diff
@@ -682,7 +682,10 @@ def _yank_url(self, what):
         else:
             flags |= QUrl.FullyEncoded
         url = QUrl(self._current_url())
-        url_query = QUrlQuery(url)
+        url_query = QUrlQuery()
+        if ';' in url.query():
+            url_query.setQueryDelimiters('=', ';')
```
Comment (lahwaacz) [inline qutebrowser/browser/commands.py]: What if the query part contains both `&` and `;` for some reason? Ampersand is standard, while semicolon is only [recommended](http://www.w3.org/TR/html401/appendix/notes.html#h-B.2.2), so I think it should be handled like this:
```python
if '&' not in url.query():
    url_query.setQueryDelimiters('=', ';')
```

Comment (HolySmoke86) [inline qutebrowser/browser/commands.py]: Good point. Actually, neither (and a few more) should be fiddled with at all, but checking for `'&' not in`, perhaps in addition, should be more robust in practice.

**Comment by The-Compiler** [comment]
Can you please add a test for 46ca91cfc04f246192197fb9035f290c6db4304d too?

**Comment by HolySmoke86** [comment]
Yeah, coming in

**Comment by The-Compiler** [comment]
Thanks!

**Pull request merged**